src/phoenix/config.py

```diff
@@ -127,4 +131,34 @@ def get_env_collector_endpoint() -> Optional[str]:
 
 
 def get_env_project_name() -> str:
-    return os.getenv(ENV_PHOENIX_PROJECT_NAME) or "default"
+    return os.getenv(ENV_PHOENIX_PROJECT_NAME) or DEFAULT_PROJECT_NAME
+
+
+def get_env_span_storage_path() -> Path:
+    """
+    Get the directory for storing traces.
+    """
+    return Path(os.getenv(ENV_SPAN_STORAGE_PATH) or (get_working_dir() / "storage"))
+
+
+def get_env_span_storage_type() -> Optional["SpanStorageType"]:
+    """
+    Get the type of span storage to use.
+    """
+    if not (env_type_str := os.getenv(ENV_SPAN_STORAGE_TYPE)):
+        return None
+    try:
+        return SpanStorageType(env_type_str.lower())
+    except ValueError:
+        raise ValueError(
+            f"⚠️ Invalid span storage type value `{env_type_str}` defined by the "
+            f"environment variable `{ENV_SPAN_STORAGE_TYPE}`. Valid values are: "
+            f"{', '.join(t.value for t in SpanStorageType)}."
+        )
+
+
+class SpanStorageType(Enum):
+    TEXT_FILES = "text-files"
+
+
+DEFAULT_PROJECT_NAME = "default"
```

Comment:
kinda hidden down here :)

Reply:
yea, it got thrown down here by IntelliJ's refactoring, which It turns out to be fine because it's very different from all the other groups of constants in this module, like ports, host, etc.